core/subsurface-qt/SettingsObjectWrapper.h

```diff
@@ -14,39 +14,6 @@
  * and QWidget frontends. This class will be huge, since
  * I need tons of properties, one for each option. */
 
-class DiveComputerSettings : public QObject {
-	Q_OBJECT
-	Q_PROPERTY(QString vendor READ dc_vendor WRITE setVendor NOTIFY vendorChanged)
-	Q_PROPERTY(QString product READ dc_product WRITE setProduct NOTIFY productChanged)
-	Q_PROPERTY(QString device READ dc_device WRITE setDevice NOTIFY deviceChanged)
-	Q_PROPERTY(QString device_name READ dc_device_name WRITE setDeviceName NOTIFY deviceNameChanged)
-	Q_PROPERTY(int download_mode READ downloadMode WRITE setDownloadMode NOTIFY downloadModeChanged)
```

Comment:
Well the other form was ok in #1378, not a single comment there. apart from that, I hate long lines. but I have changed them all, since this is not an important issue for me.